.github/workflows/update-gha.yml

```diff
@@ -0,0 +1,30 @@
+name: GitHub Actions Version Updater
+# <https://github.com/marketplace/actions/github-actions-version-updater>
+
+# Controls when the action will run.
+on:
+  # can be used to run workflow manually
+  workflow_dispatch:
+  schedule:
+    # Automatically run on 07:26 UTC every Monday
+    - cron:  '26 7 * * 1'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # [Required] Access token with `workflow` scope.
+          token: ${{ secrets.WORKFLOW_SECRET }}
+
+      - name: Run GitHub Actions Version Updater
+        uses: saadmk11/github-actions-version-updater@v0.7.2
+        with:
+          # [Required] Access token with `workflow` scope.
+          token: ${{ secrets.WORKFLOW_SECRET }}
+          committer_username: 'Brandon T. Willard'
+          committer_email: 'brandonwillard@users.noreply.github.com'
```

Comment:
We should make this something more generic (e.g. an Aesara user), if we can.

Reply:
For security we'd probably want to create a [machine account](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements).

> You must be a human to create an Account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine accounts:

Reply:
It sounds like we should be able to use the `aesara.devs@gmail.com` email.

Reply:
Unless you want it to be no-reply?

Reply:
You can use `aesara-machine-user` (and `aesara-machine-user@users.noreply.github.com`) now.

Reply:
Done!